CDCgov/prime-reportstream
# CRON sanity checks settings before checking

Fixes #6367 Reduce the number of Receiver Checks that always fail in the Receiver Check Dashboard. These come from the CRON job that runs every 2hrs.

Checks:
- the customerStatus is Active.
- there is a transport defined in the Setting.
- the transport is of a type that the checker can handle sftp right now

Todo:
- On Staging, no real checks should run, but simulated checks could fill in the database so the Dashboard page has something to display.

How to test:
 1. Before getting this change, change the frequency of the CRON from once at 8am to every hour in `prime-router/local.settings.json` e.g. (`  "REMOTE_CONNECTION_CHECK_SCHEDULE": "0 0 * * * *"`)
 2. Let run for a few hours to see ALL the settings are running using the Dashboard page.
 3. Get and run these changes. 
 Observe: only Receiver Settings that are run are Active and using SFTP transport.
 
<img width="300" alt="image" src="https://user-images.githubusercontent.com/74203452/185172214-960ee9e4-b442-43c6-a33d-a39ca1744c77.png">
Only some checks that were running (left most slices) are still running after change applied (right most slices).
